### ISSUE TITLE
Using segmentation standalone?

### ISSUE THREAD

**Issue by cnk113** [haotianteng/Xron#2]
Hi,

I was wondering if it was possible to use the NHMM as a generic signal-basecaller alignment?
I'm only doing it on DNA w/o modifications and want to use this in place of nanopolish which I assume the NHMM should still be better for?
I guess I was wondering if the model would be required here?

**Comment by haotianteng** [comment]
>I was wondering if it was possible to use the NHMM as a generic signal-basecaller alignment?

Yes! The goal of the NHMM is to deliver a better alignment algorithm by combining the basecaller's output (move table), so there will be very few long-range alignment error, and the NHMM will only need to correct local alignment (cause the move table given by basecaller is not a exact segmentation). So in short, it's totally possible to extend NHMM to DNA and other cases (such as DNA modification).

>I'm only doing it on DNA w/o modifications and want to use this in place of nanopolish which I assume the NHMM should still be better for? 

If you only want to get a segmentation, I believe the NHMM would be a pretty good alternative for Nanopolish. 

>I guess I was wondering if the model would be required here?

I am not sure what "model" you are referring to. If you refer to the basecaller model, then it's still required, as a move table from the basecaller output is used as an initial skeleton of the NHMM.
